src/clover/__main__.py

```diff
@@ -962,10 +952,10 @@ def main(  # pylint: disable=too-many-locals, too-many-statements
         logger.info("Scenario parameters valid.")
 
         logger.info("Loading grid profile.")
-        grid_profile = grid.load_grid_profile(
+        grid_profiles = grid.load_grid_profile(
             auto_generated_files_directory, logger, scenario
         )
-        logger.info("Grid '%s' profile successfully loaded.", scenario.grid_type)
+        logger.info("Grid '%s' profile successfully loaded.", scenario.grid_types)
```

Comment:
:art: Cosmetic, but we want to update the logging information here:
```diff
- logger.info("Grid '%s' profile successfully loaded.", scenario.grid_types)
+ logger.info("Grid  profile(s) %s successfully loaded.", ", ".join([f"'{grid_type}'" for grid_type in scenario.grid_types]))
```

At the minute, what we'd get outputted would look something like
```bash
Grid '['edl', 'dieselg']' profile successfully loaded."
```
whereas we want it to look more like
```bash
Grid profile(s) 'edl', 'dieselg' successfully loaded.
```